File: psuedocode/activities/problem_1/dna_to_rna.py
import sys

def dna_to_rna(s):
    ss = ""

    for i in s:
        if i == "T":
            ss += "U"
        else:
            ss += i
    return ss

s = sys.stdin.readline().strip()

while s != "":
    print(dna_to_rna(s))
    s = sys.stdin.readline().strip();

File: psuedocode/activities/problem_1/dna_to_rna.cc
#include <iostream>
#include <string>

using namespace std;

string dna_to_rna(string const& s){
  string ss = s;

  for (int i = 0; i < s.length(); i++){
    if (s[i] == 'T'){
      ss[i] = 'U';
    }
    else{
      ss[i] = s[i];
    }
  }
  return ss;
}


int main(){
  string s;

  while (cin >> s){
    cout << dna_to_rna(s) << endl;
  }
  return 0;
}
